util/execdetails/execdetails.go

```diff
@@ -422,6 +433,9 @@ func (crs *CopRuntimeStats) String() string {
 		buf.WriteString(fmt.Sprintf("%v_task:{time:%v, loops:%d", crs.storeType, FormatDuration(procTimes[0]), totalLoops))
 		if isTiFlashCop {
 			buf.WriteString(fmt.Sprintf(", threads:%d}", totalThreads))
+			if !totalTiFlashScanContext.Empty() {
+				buf.WriteString(fmt.Sprintf(", tiflash_scan:{dmfile:{total_scanned_packs:%d, total_skipped_packs:%d, total_scanned_rows:%d, total_skipped_rows:%d, total_rough_set_index_load_time: %dms, total_dmfile_read_time: %dms}, total_create_snapshot_time: %dms}", totalTiFlashScanContext.totalScannedPacksInDmfile, totalTiFlashScanContext.totalSkippedPacksInDmfile, totalTiFlashScanContext.totalScannedRowsInDmfile, totalTiFlashScanContext.totalSkippedRowsInDmfile, totalTiFlashScanContext.totalRoughSetIndexLoadTimeInNs/1000000, totalTiFlashScanContext.totalDmfileReadTimeInNs/1000000, totalTiFlashScanContext.totalCreateSnapshotTimeInNs/1000000))
```

Comment:
some duplicate code here, how about adding `String` method for `TiFlashScanContext`?

Reply:
Good suggestion! I add String() method for TiFlashScanContext now!